src/components/homepage/Homepage.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSelector } from "react-redux";
+
+export interface HomeProps {} const HomePage = (props:HomeProps) => {
+
+    const data:any = useSelector((state) => state);
```

Comment:
please add check if data is null
let's create a state for redux storage to check if data is loaded. more info here - https://redux.js.org/tutorials/essentials/part-5-async-logic